eng/pipelines/templates/steps/build.yml

```diff
@@ -3,6 +3,19 @@ parameters:
   ServiceDirectory: not-specified
 
 steps:
+  - task: PowerShell@2
+    displayName: Prep template pipeline for release
+    condition: and(succeeded(),eq(variables['TestPipeline'],'true'))
+    inputs:
+      pwsh: true
+      workingDirectory: $(Build.SourcesDirectory)
+      filePath: eng/scripts/SetTestPipelineVersion.ps1
+
+  - pwsh: |
```

Comment:
What is this step for?

Reply:
Am going to remove that. I was trying to do it with the existing tools.